x/msg_authorization/client/cli/tx.go

```diff
@@ -0,0 +1,36 @@
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/x/msg_authorization/internal/types"
+	"github.com/spf13/cobra"
+)
+
+// GetTxCmd returns the transaction commands for this module
+func GetTxCmd(cdc *codec.Codec) *cobra.Command {
+	AuthorizationTxCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "Authorization transactions subcommands",
```

Comment:
Please add a `Long` text, even if it is the same as `Short`

Reply:
Sure, will ensure when I start working on client